apps/dashboard/src/routes/(app)/courses/[id]/certificates/+page.svelte

```
@@ -18,9 +18,15 @@
 
   let errors = $state<Record<string, string>>({});
 
-  const certificateActiveTab = $derived.by(() => {
-    const tabInParam = page.url.searchParams.get('tab');
-    return tabInParam === 'settings' ? 'settings' : 'design';
+  function normalizeCertificateTab(tab: string | null) {
+    return tab === 'settings' ? 'settings' : 'design';
+  }
+
+  // replaceState does not update page.url; local state handles tab clicks, this resyncs on navigation.
+  let certificateActiveTab = $state(normalizeCertificateTab(page.url.searchParams.get('tab')));
```

ESLint

[error] Prefer using writable $derived instead of $state and $effect

(svelte/prefer-writable-derived)